scripts/generate.rb

```diff
@@ -4,7 +4,11 @@
 
 require 'yaml'
 
-holidays_detailed = YAML.load_file('./holiday_jp/holidays_detailed.yml')
+if YAML.respond_to?(:unsafe_load_file)
+  holidays_detailed = YAML.unsafe_load_file('./holiday_jp/holidays_detailed.yml')
+else
+  holidays_detailed = YAML.load_file('./holiday_jp/holidays_detailed.yml')
+end
```

Comment:
https://github.com/holiday-jp/holiday_jp-ruby/blob/master/lib/holiday_jp/holidays.rb#L11-L15

I used above as a reference for the source code.